apps/wpcom-block-editor/README.md

```diff
@@ -0,0 +1,3 @@
+# WPCOM block editor
```

Comment:
Note that https://github.com/Automattic/wp-calypso/pull/32242 has been merged so you can remove the changes on this file :)